### DISCUSSION TITLE
Certificate Private Key and PFX Password to Store on the KV

### DISCUSSION THREAD

**Discussion by pir8g33k** [polymind-inc/acmebot#574] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Can we include when we generate a certificate and it will include or store the private key as well into a key vault and also when new export the certificate with pfx what is the default password?

**Describe the solution you'd like**
for the PFX at least we cann still use these as backup and export to other services as well like azure enterprise application proxy and iis

**Describe alternatives you've considered**

**Additional context**

**Comment by shibayan** [comment, marked as answer]
PFX exported from Key Vault do not have passwords attached, which is the behavior of the Key Vault side.

https://learn.microsoft.com/en-us/azure/key-vault/certificates/how-to-export-certificate?tabs=azure-cli#export-stored-certificates
